wandb/apis/public/projects.py

```diff
@@ -202,10 +197,22 @@ def __init__(
             attrs: The attributes of the project.
         """
         super().__init__(dict(attrs))
+        self._is_loaded = False if not attrs else True
         self.client = client
         self.name = project
         self.entity = entity
 
+    def _load(self):
+        variable_values = {"project": self.name, "entity": self.entity}
+        try:
+            response = self.client.execute(self.QUERY, variable_values)
+            project = response["project"]
```

Comment:
should be outside the `try-except`, but you can also inline it on the `self._attrs` line, right?